src/app/game/game-intermediate-result/game-drawing-display/game-drawing-display.component.ts

```diff
@@ -28,4 +37,11 @@ export class GameDrawingDisplayComponent {
         return '';
     }
   }
+
+  greenText(): string {
+    if (this.hasCorrectGuess) {
+      return 'green-text';
+    }
+    return '';
+  }

```

Comment:
struggled abit to understand that this was css related

Reply:
Good feedback! Will fix